app/js/components/labOrderEntry/DraftOrder/DraftList.jsx

```diff
@@ -1,20 +1,59 @@
 import React from 'react';
 import shortid from 'shortid';
 import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faExclamationTriangle, faCheckSquare } from '@fortawesome/free-solid-svg-icons';
+
 
 export class DraftList extends React.Component {
+  static toggleIcon(order) {
+    let orderUrgencyIcon;
```

Comment:
this is not really necessary especially if we had to do this everywhere in the app to alternate styles, you can do this in any of two ways
1. hold that value in state anyway and then setState when it's toggled which would trigger a rerender
2. `toggleDraftLabOrdersUgency` dispatches an action which causes redux state change and then your component rerenders

but I see you'd still need to toggle between styles so just install and use the [classnames](https://www.npmjs.com/package/classnames) package then use the `urgency` prop in store to hold the styles

Reply:
Fixed